### DISCUSSION TITLE
is_child_of / has_parent showdown

### DISCUSSION THREAD

**Discussion by oliviahobai** [CommonCoreOntology/CommonCoreOntologies#576] [Announcements]
Per issue #[490](https://github.com/CommonCoreOntology/CommonCoreOntologies/issues/490), the is_parent_of property has two inversions: is_child_of and has_parent.
There does not seem to be a major or necessary difference in meaning that would ask to keep both.
Additional opinions are welcome and appreciated:

**Comment by alanruttenberg** [comment]
related: #560

**Reply by oliviahobai** [reply]
I'm moved to keep only "is child of", and make "has parent" an equivalent class in a new domain ontology as @cameronmore proposed last month. Alan, do you have a preference?
